.github/workflows/CI.yml

```diff
@@ -1,10 +1,19 @@
 name: CI
+
 on:
-  - push
-  - pull_request
+  push:
+    branches: [master]
```

Comment:
Perhaps we should also run on tags?
```suggestion
    branches: [master]
    tags: [v*]
```

Reply:
I don't mind. But I don't think it is necessary - it is run for every commit pushed to the master branch anyway. I think the `tags` trigger is mainly useful for building release-specific documentation.

Reply:
We'll want that eventually anyways, once we have documentation. I'll leave it up to you whether to add it.

Reply:
For documentation we might want to add a separate action anyway, IMO that's cleaner (and e.g. allows more fine-tuned triggers) :shrug:

Reply:
I don't add the trigger in this PR, we can add it later if it seems useful.